code_schemes/s02e01.json

```diff
@@ -0,0 +1,178 @@
+{
+  "SchemeID": "Scheme-eea92621",
+  "Name": "S02E01",
```

Comment:
Would it make sense to call this LEAP S02E01?

Reply:
Yup that would be useful. Updated